Doc/library/typing.rst

```diff
@@ -459,6 +502,37 @@ The module defines the following classes, functions and decorators:
           except KeyError:
               return default
 
+.. class:: Protocol(Generic)
+
+   Base class for protocol classes. Protocol classes are defined as::
+
+      class Proto(Protocol):
+          def meth(self) -> int:
+              ...
+
+   Such classes are primarily used with static type checkers that recognize
+   structural subtyping (static duck-typing), for example::
+
+      class C:
+          def meth(self) -> int:
+              return 0
+
+      def func(x: Proto) -> int:
+          return x.meth()
+
+      func(C())  # Passes static type check
+
+   See :pep:`544` for details. Protocol classes decorated with
+   :func:`runtime_checkable` act as simple-minded runtime protocols that
```

Comment:
Is there no `:decorator:` thing you can use? Also maybe insert "(described later)" since this isn't the full description of runtime_checkable, just a summary of features?

Reply:
> Is there no `:decorator:` thing you can use?

I think unfortunately no. At least I didn't find `:deco:` or anything similar (this would be something good to have).